ui-admin/src/portal/dashboard/widgets/WebsiteWidget.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button } from 'components/forms/Button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEye, faGear, faImage, faPalette, faPencil } from '@fortawesome/free-solid-svg-icons'
+import React from 'react'
+import { Portal } from '@juniper/ui-core'
+import { DropdownButton } from 'study/participants/survey/SurveyResponseView'
+import { useNavigate } from 'react-router-dom'
+import { siteContentPath, siteMediaPath } from 'portal/PortalRouter'
+import { InfoCard, InfoCardBody, InfoCardHeader } from 'components/InfoCard'
+
+export const WebsiteWidget = ({ portal }: { portal: Portal }) => {
+  const livePortalUrl = portal.portalEnvironments.find(env =>
+    env.environmentName === 'live')?.portalEnvironmentConfig.participantHostname
+
+  return (
+    <InfoCard classNames={''}>
+      <InfoCardHeader>
+        <div className="d-flex align-items-center justify-content-between w-100">
+          <span className="fw-bold">Website</span>
+          <div className="d-flex">
+            <Button onClick={() => {
+              const url = livePortalUrl?.startsWith('http') ? livePortalUrl : `https://${livePortalUrl}`
+              window.open(url, '_blank')
+            }}
+            tooltip={'Preview live website'}
+            variant="light" className="border m-1">
+              <FontAwesomeIcon icon={faEye} className="fa-lg"/> Preview
+            </Button>
+            <CustomizeWebsiteDropdown portal={portal}/>
+          </div>
+        </div>
+      </InfoCardHeader>
+      <InfoCardBody>
+        <div className="d-flex">
+          <div style={{ minHeight: '200px' }}>
+            { livePortalUrl ?
+              <img className="border rounded-3"
+                style={{ maxWidth: '100%', height: 'auto' }}
+                src={`https://api.urlbox.io/v1/A8hJem0DKIAE8mDm/png?url=${livePortalUrl}`}
```

Comment:
my trial expired, so the screenshots don't work anymore =/ but it's not expensive ($50-100/month)

there might be cheaper alternatives, i can put a placeholder button here for now. i don't think this should block merging

Reply:
FWIW I could get 90% of the way there (with a bit of weirdness) with just CSS scaling properties. I also wonder if we used something like https://github.com/bubkoo/html-to-image we could get it to work.